apps/meteor/app/statistics/server/lib/SAUMonitor.ts

```diff
@@ -321,7 +321,7 @@ export class SAUMonitorClass {
 		logger.info('[aggregate] - Aggregating data.');
 
 		const date = new Date();
-		date.setDate(date.getDate() - 0); // yesterday
+		date.setDate(date.getDate() - 1); // yesterday
```

Comment:
```suggestion
```
The idea in this line is just to make clear that this date is set like this so that we can aggregate the data from "yesterday" (this date is a superior constraint, as you can check in `dailySessionsOfYesterday`). So I think the approaches we can choose here are either to remove this line or keep it as it is so that the engagement dashboard continues to work
Honestly, I think it's confusing as it is hahah so maybe we should remove it

Reply:
Okay, got it. I was thinking that was wrong since `getDate() - 0` isn't yesterday 😆